packages/native/test/lib/ElementAssertion.test.tsx

```diff
@@ -1,13 +1,40 @@
 import { AssertionError, expect } from "@assertive-ts/core";
-import { render } from "@testing-library/react-native";
+import { fireEvent, render } from "@testing-library/react-native";
+import { useState, useCallback } from "react";
 import {
   View,
   TextInput,
   Text,
+  Modal,
+  Button,
 } from "react-native";
 
 import { ElementAssertion } from "../../src/lib/ElementAssertion";
 
+const SimpleToggleText: React.FC = () => {
+  const [isVisible, setIsVisible] = useState(true);
+
+  const handleToggle = useCallback((): void => {
+    setIsVisible((prev: boolean) => !prev);
+  }, []);
+
+  return (
+    <View>
+      <Text
+        testID="textElement"
+        style={{ display: isVisible ? "flex" : "none" }}
+      >
+        {"Toggle me!"}
+      </Text>
+      <Button
+        testID="toggleButton"
```

Comment:
Same on this one. The `byRole` query is preferred in most cases, and for this element, we should be able to do `getByRole("button", { name: "Toggle Text" })` instead.